gulpfile.js

```diff
@@ -87,11 +89,15 @@ checkCommands(yargs, argv, 1);
 var isMinificationNeeded = (argv.minify == 'True');
 var frontendDependencies = manifest.dependencies.frontend;
 var cssFilesPath = [];
+var jsFilesPath = [];
 var fontFolderPath = [];
 var cssBackgroundPath = [];
 var generatedCssTargetDir = path.join(
```

Comment:
Can we compute targetDirPrefix first, and then join that with 'css' or 'js' as needed? Seems simpler to have just one location for the conditional logic.